Makefile.am

```diff
@@ -7,6 +7,16 @@ MAINTAINERCLEANFILES = test-driver \
 	config.sub depcomp install-sh ltmain.sh m4/* missing \
 	depcomp ylwrap
 
+if AIX
+SHLIBEXT=a
```

Comment:
I guess it would be better readably if the statements inside the if or else blocks are indented, like it is done below.

Reply:
Ah ok, then leave it as is.